data/guis/gitgui.yml

```diff
@@ -1,12 +0,0 @@
----
-name: "Gitgui"
-project_url: "https://github.com/jjustra/gitgui"
```

Comment:
[That repository](https://github.com/jjustra/gitgui) is not marked as archived. Therefore it will require more than a PR without explanation to remove it.